src/core/models/model_list.py

```diff
@@ -40,6 +40,8 @@ def d_state(self, t: float, state: State) -> Dict[str, State_Type]:
         # craft to origin
         r_co = np.array([state.x, state.y, state.z])
         # moon to origin
+        
+        # round t to the 10s place
```

Comment:
Where does this happen?

Reply:
Yeah sorry, I did a clown move and deleted the changes temporarily to compare with the original runtime, but then forgot to change them back when I did a commit after. Should be there now though